### ISSUE TITLE
Allow .MSYM directories as symbols in Distribute task for DevOps Classic Release pipeline

### ISSUE THREAD

**Issue by AlphaNERD-** [microsoft/appcenter#2370]
**Describe the solution you'd like**
I have this little problem that the App Center requires me to send the MSYM directory in order to fully symbolicate errors and crashes. I wanna do that within the classic release pipeline. However the App Center Distribute task in the classic release pipeline does not seem to support submitting symbols in the MSYM directory. For Android it only supports the mapping.txt file which i can't create because i use Xamarin.Android and ProGuard cannot obfuscate Xamarin.Android apps. Can't have a file for deobfuscation if my app is not obfuscated in the first place.

Basically i want to be able to send an MSYM directory through the App Center Distribute task in the classic release pipeline.

**Describe alternatives you've considered**
Sending it through my build pipeline which could render my release pipeline obsolete. The YAML task allows me to configure a symbols path and doesn't seem to be restricted on what i could send for a given symbol type. Or i could make something custom using your REST API. Wouldn't be the first time i fixed an oversight in your tools by myself.

**Comment by AlphaNERD-** [comment]
This issue is not stale at all. This issue is still present. By the way i later found out that i cannot upload MSYM files through your web interface so we should change that issue to "Add support for .MSYM directories" period.

**Comment by AlphaNERD-** [comment]
It's still not stale, it's still a pain in the butt.

**Comment by AlphaNERD-** [comment]
NO! NO! Bad bot! Shoo!

**Comment by AlphaNERD-** [comment]
Nope. Still open.

**Comment by AlphaNERD-** [comment]
Nope, we still need that.
